ci-operator/jobs/openshift/csi-driver-manila-operator/openshift-csi-driver-manila-operator-master-presubmits.yaml

```diff
@@ -75,6 +75,70 @@ presubmits:
         secret:
           secretName: registry-pull-credentials
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
```

Comment:
Could you make the job as optional (preferably in a separate commit) until we stabilize the job?